ivy.el

```diff
@@ -58,94 +58,128 @@
-  "Face used by Ivy for a match required prompt.")
+  "Face used by Ivy for a match required prompt."
+  :group 'ivy-faces)
 
 (defface ivy-subdir
   '((t :inherit dired-directory))
-  "Face used by Ivy for highlighting subdirs in the alternatives.")
+  "Face used by Ivy for highlighting subdirs in the alternatives."
+  :group 'ivy-faces)
 
 (defface ivy-modified-buffer
   '((t :inherit default))
-  "Face used by Ivy for highlighting modified file visiting buffers.")
+  "Face used by Ivy for highlighting modified file visiting buffers."
+  :group 'ivy-faces)
 
 (defface ivy-modified-outside-buffer
   '((t :inherit default))
-  "Face used by Ivy for highlighting file visiting buffers modified outside Emacs.")
+  "Face used by Ivy for highlighting file visiting buffers modified outside Emacs."
+  :group 'ivy-faces)
 
 (defface ivy-remote
   '((((class color) (background light))
      :foreground "#110099")
     (((class color) (background dark))
      :foreground "#7B6BFF"))
-  "Face used by Ivy for highlighting remotes in the alternatives.")
+  "Face used by Ivy for highlighting remotes in the alternatives."
+  :group 'ivy-faces)
 
 (defface ivy-virtual
   '((t :inherit font-lock-builtin-face))
-  "Face used by Ivy for matching virtual buffer names.")
+  "Face used by Ivy for matching virtual buffer names."
+  :group 'ivy-faces)
 
 (defface ivy-action
   '((t :inherit font-lock-builtin-face))
-  "Face used by Ivy for displaying keys in `ivy-read-action'.")
+  "Face used by Ivy for displaying keys in `ivy-read-action'."
+  :group 'ivy-faces)
 
 (defface ivy-highlight-face
   '((t :inherit highlight))
-  "Face used by Ivy to highlight certain candidates.")
+  "Face used by Ivy to highlight certain candidates."
+  :group 'ivy-faces)
 
 (defface ivy-prompt-match
   '((t :inherit ivy-current-match))
-  "Face used by Ivy for highlighting the selected prompt line.")
+  "Face used by Ivy for highlighting the selected prompt line."
+  :group 'ivy-faces)
 
 (defface ivy-separator
   '((t :inherit font-lock-doc-face))
-  "Face for multiline source separator.")
+  "Face for multiline source separator."
+  :group 'ivy-faces)
 
 (defface ivy-grep-info
   '((t :inherit compilation-info))
-  "Face for highlighting grep information such as file names.")
+  "Face for highlighting grep information such as file names."
+  :group 'ivy-faces)
 
 (defface ivy-grep-line-number
   '((t :inherit compilation-line-number))
-  "Face for displaying line numbers in grep messages.")
+  "Face for displaying line numbers in grep messages."
+  :group 'ivy-faces)
+
+(defface ivy-variable-documentation
+  '((t :inherit font-lock-comment-face))
+  "Face for displaying lisp documentation."
+  :group 'ivy-faces)
+
+(defface ivy-application-name
+  '((t :inherit font-lock-builtin-face))
+  "Face for displaying executable names."
+  :group 'ivy-faces)
+
+(defface ivy-key-bindings
+  '((t :inherit font-lock-builtin-face))
+  "Face for displaying key bindings."
+  :group 'ivy-faces)
```

Comment:
As previously mentioned, I think the existing face `counsel-key-binding` already fits the bill.